.github/ISSUE_TEMPLATE.md

```diff
@@ -30,6 +35,9 @@ modinfo spl | grep -iw version
 
 ### Include any warning/errors/backtraces from the system logs
 <!-- 
+*IMPORTANT* - Please mark logs or Github will not display them correctly. 
```

Comment:
Maybe you're right, but i think it's obvious enough with `logs` only.

Maybe it's better to add this info to header?
`### Include any warning/errors/backtraces from the system logs or commands output`